styles/style.css

```diff
@@ -0,0 +1,43 @@
+html {
+    font-size: 10px; 
+    font-family: 'Didact Gothic', Tahoma, sans-serif;
```

Comment:
i think you need to import this font in order to use it

Reply:
I replaced it by Verdana